src/backend/joanie/core/templates/issuers/certificate.html

```diff
@@ -51,9 +51,9 @@ <h1 class="header__title">{% translate "Attestation of achievement" %}</h1>
                 <p><sup>*</sup>{% translate "MOOC: Massive Open Online Course" %}</p>
                 <p>
                     <strong>
-                    {% blocktranslate trimmed with organization=organization.name %}
+                    {% blocktranslate trimmed with organizations=organizations|list_key:'name'|join_and %}
```

Comment:
Is this filter `join_and` will be translated to french ?  
In order to get "et" not "and" before the last organization, when we are in a french context for example.

Should we override the `join_and` filter ?

Reply:
Yup this tag is internationalized.

Reply:
Magnifique